### ISSUE TITLE
quick2D fails with dimensionality error.

### ISSUE THREAD

**Issue by darienmorrow** [wright-group/WrightTools#587]
I have tried three ways to get quick2D to plot some 2D freqs. All have failed. One error message is:
```
  File "/home/darien/source/WrightTools/WrightTools/artists/_base.py", line 363, in pcolor
    raise wt_exceptions.DimensionalityError(2, data.ndim)

DimensionalityError: (DimensionalityError(...), 'dimensionality must be 2 (recieved 1)')
```
Note, quick2D works when I plot yaxis=1 and xaxis=0 (which plots ND wheel vs w2).

Minimum working example:
```
import WrightTools as wt
d = wt.data._pycmds.from_PyCMDS(r'early.data')
d.signal_ratio.signed = True
#wt.artists.quick2D(d, channel='signal_ratio', yaxis=1, xaxis=2, autosave=True)
#wt.artists.quick2D(d, channel='signal_ratio', yaxis='w2', xaxis='w1__e__wm', autosave=True)
wt.artists.quick2D(d, channel='signal_ratio', yaxis='w2', xaxis='w1=wm', autosave=True)
```
[early.zip](https://github.com/wright-group/WrightTools/files/1938037/early.zip)

Please rename issue with better name if fitting.

**Comment by darienmorrow** [comment]
I think the problem is with `chop`.
`out = d.chop('w1=wm', 'w2')` yields a collection in which each data object has dimension (36,1), instead of the desired (36,36).

**Comment by ksunden** [comment]
Chop has some issues with equality expressions, I believe (and perhaps expressions in general)

Can you try transforming the data object such that the axis is just wm or w1,not wm=w1?

**Comment by ksunden** [comment]
The problem here is kind of two fold (and I've seen it before, though I thought I had fixed it, apparently not):

from_PyCMDS is creating 'd1' with a shape of 4x36x1, despite the fact that it is a scanned axis which should just have a shape of 4x1x1.
This is causing problems in chop (which Ideally chop would be able to handle itself.)

So, in summary, from_PyCMDS is doing something which is not intended, but shouldn't be illegal.
Chop is not set up to handle that kind of data at this time (and will require a fair bit more thinking to get there, though it is doable)

**Comment by ksunden** [comment]
This was fixed by #596
